core/ir/aarch64/codec.txt

```diff
@@ -212,1135 +212,1153 @@ x----------xxxxx----------------  wx16       # W/X register (or WZR/XZR)
 # {de,en}code_opnds_OPNDSET in "codec.c", that handles all the operands together.
 # This is used, for example, when the number of operands varies.
 
+################################################################################
+
+# Condition register access field: nzcv_flag.
+
+# This field tells the decode genarator to set NZCV flag register usage for the
+# instruction, i.e. whether the instruction can read from the register, write
+# to the register or both.
+
+# There are 6 states for nzcv_flag. The field should appear between the
+# instruction pattern and the opcode as shown above. The flag states are:
+# n No read or write
+# r Read only
+# w Write only
+# rw or wr Read and write
+# er Explicit/direct read (currently only MRS instruction)
```

Comment:
Maybe mention that these require companion code in codec.c to set the instr_t.eflags field

Reply:
Done.